internal/config/config.go

```diff
@@ -46,6 +46,7 @@ type SauceConfig struct {
 	Retries     int               `yaml:"retries,omitempty" json:"-"`
 	Visibility  string            `yaml:"visibility,omitempty" json:"-"`
 	LaunchOrder LaunchOrder       `yaml:"launchOrder,omitempty" json:"launchOrder,omitempty"`
+	Workload    string            `yaml:"workload,omitempty" json:"workload,omitempty"`
```

Comment:
Wondering if that makes sense to but this at this level.
Maybe the suite level is more correct (and then rely on `defaults` if we want a global value).

Would that be an expected usage to get multiple workload kinds in the same `config.yml` ?

Reply:
Great question! We do allow the user to specify the image on a suite level, so in _theory_ the user could have different suites with different images that each perform a different workload type. If we don't support this, the user has to create a separate configuration files. Neither is better or worse. Having the option allows the user to combine multiple workloads in a single config or spread it out across configs. Without the suite level option, the user has to use multiple configs.

@waggledans would you also like to weigh in on this?

Reply:
If I understand it correctly,  the `Workload` should be at the `suite` level, right next to the `image`/`entrypoint` spec.
If there is a use case to specify the `workload` for all suites, then sure, we can add it here as well, so we'd have both options.

Reply:
@FriggaHel If you could make that change, that'd be great 💪

Reply:
Yup, I will do that change ! I will shift it to the `defaults` field.